code/modules/admin/admin_verbs.dm

```diff
@@ -918,3 +919,13 @@ var/global/list/admin_verbs_mod = list(
 
 		if("Cancel")
 			return
+
+/client/proc/force_reload_theme_css()
```

Comment:
why is this on `/client` and not `/datum/admins` if we're just assuming src is `/datum/admins` anyway??

Reply:
Well, mainly because it's a clientside thing. It doesn't really makes sense to me to put it under the admin datum and end up dereferencing a bunch of vars to get the client, while I could just call it on the client. Idk.

Reply:
> That doesn't make any sense. You could just call `get_client()` without casting if that was a concern, but even then, `get_client()` will just return `src`. This is pretty strange code.

Yeah. I think you're right. I probably should have re-read that thing more closely...